src/Geometry/PUMLReader.cpp

```diff
@@ -363,7 +363,7 @@ void seissol::PUMLReader::getMesh(const PUML::TETPUML &puml)
 
 			int bcCurrentFace = (boundaryCond[i] >> (j*8)) & 0xFF;
 			int faultTag = bcCurrentFace;
-			if (bcCurrentFace > 6) {
+			if (bcCurrentFace > 8) {
```

Comment:
This change might break some existing setups. You need to inform the guys and the girls.

Reply:
Maybe we should make this change more drastic, i.e. bcCurrentFace > 32, 64 or 128. Then there is some space for future bcs.